test/nightly/azureResourceGraph.test.ts

```diff
@@ -5,12 +5,11 @@
 
 import { ContainerDefinition, CosmosClient, DatabaseDefinition, Resource } from '@azure/cosmos';
 import * as assert from 'assert';
-import { CosmosDBManagementModels } from 'azure-arm-cosmosdb';
 import * as vscode from 'vscode';
 import { randomUtils } from '../../extension.bundle';
 import { longRunningTestsEnabled, testUserInput } from '../global.test';
 import { getConnectionString } from './getConnectionString';
-import { client, resourceGroupsToDelete, testAccount } from './global.resource.test';
+import { accountList, api, resourceGrouList, resourceGroupsToDelete, testAccount } from './global.resource.test';
 
 suite('Graph action', async function (this: Mocha.Suite): Promise<void> {
```

Comment:
Could you add `require(./azureResourceCreateAccount.test.ts)`?  I think that this should worked based off this example https://stackoverflow.com/a/32934285

I'm nervous if the tests ever run out of order or we have a name that comes before the account creating test, then the accounts won't have been created.

Reply:
I add require(./azureResourceCreateAccount.test.ts) and this file will run twice.

Note: I will create account in the "global.resource.test.ts" file. I think this problem can be solved.